apps/homepage/src/app/_shared/header/header.module.scss

```diff
@@ -0,0 +1,18 @@
+.main{
```

Comment:
Looks like this is the same file as this one apps/homepage/src/app/[lang]/_shared/header/header.module.scss ?

Reply:
Yes, i just noticed that when i merged my branch with main i duplicated the _shared folder outside of [lang], ill delete the folder